test/Broadway/CommandHandling/SimpleCommandBusTest.php

```diff
@@ -100,9 +103,13 @@ public function it_should_still_handle_commands_after_exception()
         $this->commandBus->dispatch($command2);
     }
 
+    /**
+     * @param $expectedCommand
+     * @return SimpleCommandHandler|\PHPUnit_Framework_MockObject_MockObject
```

Comment:
in this case it only returns the mock.

Reply:
can you add it as a return type hint? that's better than doc blocks.

Reply:
Ok, can we set "SimpleCommandHandler" as return type? Because this is what it should be in the end.

Reply:
yes, please